examples/notify-turn-hook.py

```
@@ -0,0 +1,220 @@
+a spoken notification via audio-notify-server. Only triggers for
+tasks that take longer than MIN_DURATION seconds.
+"""
+
+import json
+import os
+import subprocess
+import sys
+from datetime import datetime
+from pathlib import Path
+
+LOCKFILE = Path("/tmp/notify-hook.lock")
```

Ruff (0.14.6)

Probable insecure usage of temporary file or directory: "/tmp/notify-hook.lock"

(S108)